P2kDoom8/Makefile.eg1

```diff
@@ -2,7 +2,7 @@
 # Flavor: EG1, EA1 (ElfPack v1.x, ARM GCC, Linux)
 # Author: EXL, 06-Jan-2025
 
-P2K_ELF_SDK       = /home/exl/Storage/Projects/Git/MotoFanRu/P2K-ELF-SDK
+P2K_ELF_SDK       = /home/fk/Motorola/P2K-ELF-SDK
```

Comment:
It will be replaced with a general path once the general P2K-ELF-SDK is completed.

Therefore, please drop it.